operators/src/processing/raster_vector_join/non_aggregated.rs

```diff
@@ -288,6 +296,7 @@ where
         for (raster_processor, new_column_name) in
             self.raster_processors.iter().zip(&self.column_names)
         {
+            log::debug!("processing raster for new column {:?}", new_column_name);
```

Comment:
```suggestion
            log::debug!("processing raster for new column {:?}", new_column_name);
            
```

Reply:
i tried that but git thinks there is no difference